openapi/business/business.yaml

```diff
@@ -208,6 +208,183 @@ paths:
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/customers'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+                        format: uri
+                      first:
+                        type: string
+                        description: Provides a link back to the first page of results
+                        format: uri
+                      next:
+                        type: string
+                        description: The URI at which the next batch of customers can be gotten from
+                        format: uri
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Produces a list of customers
+      parameters:
+        - schema:
+            type: string
+          in: header
+          name: Authorization
+          description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
+          required: true
+        - schema:
+            type: string
+          in: query
+          name: 'filter[businesses.id]'
+          description: Return customers for the specified business location
+          required: true
+        - schema:
+            type: string
+          in: query
+          description: The cursor stores all your filters and current location in the list to allow paging over the results in smaller batches. The value will be provided in the response links.
+          name: 'page[cursor]'
+        - schema:
+            type: string
+          in: query
+          description: The maximum number of tasks you would like returned in a single batch. Use the links.next member in the response to get the remainder.
+          name: 'page[limit]'
+      security:
+        - OAuth2:
+            - customers
+      tags:
+        - Customers
+      x-lifecycle:
+        status: trustedTester
+  '/customers/{id}':
+    parameters:
+      - schema:
+          type: string
+        name: id
+        in: path
+        required: true
+    get:
+      summary: Get Customer
+      tags:
+        - Customers
+      responses:
+        '200':
+          description: OK
+          content:
+            application/vnd.api+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    $ref: '#/components/schemas/customers'
+                  links:
+                    type: object
+                    properties:
+                      self:
+                        type: string
+      operationId: get-customers-id
```

Comment:
I guess that this operation is actually to get a customer by id, is it correct ?